### PR TITLE
First version

### PR DESCRIPTION
**Pull request by katcipis**

### CODE REVIEW

File: templates/Makefile [hunk 1]
```diff
@@ -0,0 +1,45 @@
+version ?= latest
+{{project}}_img={{docker_registry}}/{{project}}:$(version)
+devimg=phogdev
```
Comment (vitorarins) [inline templates/Makefile]: oops

File: gootstrap.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+			fmt.Printf("unexpected path[%s], unable to parse it\n", templatefile)
+			os.Exit(1)
+		}
+		relativePath := applyPathTemplate(templatedPath, project)
+		outputfile := path.Join(outputdir, relativePath)
+		templateToOutput[templatefile] = outputfile
+		outputdirs[filepath.Dir(outputfile)] = struct{}{}
+	}
+
+	for outdir, _ := range outputdirs {
```
Comment (LeoCBS) [inline gootstrap.go]: if you ignoring second var.. this loop is equivalent to ```for outdir := range ...``` ?

Comment (katcipis) [reply gootstrap.go]: indeed, thanks ;-)

**Review by cadicallegari — APPROVED** [review]

**Comment by cadicallegari** [comment]
wouldn't be nice create a readme file too?

**Review by LeoCBS — APPROVED** [review]

**Pull request merged**